R/utils.R

```diff
@@ -113,20 +113,6 @@ isWholeNum <- function(x, tol = .Machine$double.eps^0.5) {
   abs(x - round(x)) < tol
 }
 
-`%OR%` <- function(x, y) {
-  if (is.null(x) || isTRUE(is.na(x)))
-    y
-  else
-    x
-}
-
-`%AND%` <- function(x, y) {
-  if (!is.null(x) && !isTRUE(is.na(x)))
-    if (!is.null(y) && !isTRUE(is.na(y)))
-      return(y)
-  return(NULL)
-}
-
 `%.%` <- function(x, y) {
   paste(x, y, sep='')
 }
```

Comment:
While we're at it, it'd be nice to get rid of this too (and use `paste0()` instead)?

Reply:
Done in b0a1a06b.